Sources/MongoSwift/WriteConcern.swift

```diff
@@ -95,10 +95,13 @@ public class WriteConcern: Codable {
     /// Initializes a new `WriteConcern`.
     /// - Throws:
     ///   - `UserError.invalidArgumentError` if the options form an invalid combination.
-    public init(journal: Bool? = nil, w: W? = nil, wtimeoutMS: Int32? = nil) throws {
+    public init(journal: Bool? = nil, w: W? = nil, wtimeoutMS: Int64? = nil) throws {
```

Comment:
The write concern spec defines wtimeoutMS as an `int64`, but libmongoc expects an `int32` here. I've updated it to just truncate if the value is too big. I think it's worth getting our API correct now so that if we ever get off of libmongoc or they update the signature, we won't need to maintain two initializers until a major version bump.

One question though: would it be preferable to just throw an error here?

Reply:
Filed [SWIFT-395](https://jira.mongodb.org/browse/SWIFT-395)